src/server/game/Entities/Player/Player.cpp

```diff
@@ -7794,7 +7794,7 @@ bool Player::IsOutdoorPvPActive()
 void Player::DuelComplete(DuelCompleteType type)
 { 
     // duel not requested
-    if (!duel)
+    if (!duel || duel->opponent || duel->initiator)
```

Comment:
Shouldn't this be
```cpp
if (!duel || !duel->opponent || !duel->initiator)
```
or am I missing something?

Reply:
Good catch!